generators/entity-client/templates/angular/src/main/webapp/app/entities/update/entity-management-update.component.spec.ts.ejs

```diff
@@ -124,6 +129,79 @@ _%>
     <%_ } _%>
         });
 <%_ } _%>
+<%_ if (relationships.filter(rel => rel.ownerSide).length > 0) { _%>
+
+        describe('Loading relationships', () => {
+            describe('loadRelationshipsOptions', () => {
+    <%_ for (const relationshipsByEntityNeedingOptions of allRelationshipsByEntityNeedingOptions) { _%>
+        <%_ const relationshipsWithCustomUniqueOptions = relationshipsByEntityNeedingOptions.filter(rel => rel.relationshipType === 'one-to-one' && rel.otherEntityName !== 'user'); %>
+        <%_ const relationshipsWithCustomSharedOptions = relationshipsByEntityNeedingOptions.filter(rel => !relationshipsWithCustomUniqueOptions.includes(rel)); %>
+        <%_ if (relationshipsWithCustomSharedOptions.length > 0) { _%>
+            <%_ const otherEntity = relationshipsByEntityNeedingOptions[0].otherEntity _%>
+
+                it('Should call <%= otherEntity.entityAngularName %> query and add missing value', () => {
+                    fakeAsync(() => {
+                        const <%= entityInstance %> : I<%= entityAngularName %> = <%- generateTestEntityPrimaryKey(primaryKey, 0) %>;
+            <%_ for (const relationship of relationshipsWithCustomSharedOptions) { _%>
+                <%_ const reference = relationship.reference _%>
+                <%_ if (relationship.collection) { _%>
+                        const <%= reference.name %> : I<%= otherEntity.entityAngularName %>[] = [<%- generateTestEntityPrimaryKey(otherEntity.primaryKey) %>];
+                <%_ } else { _%>
+                        const <%= reference.name %> : I<%= otherEntity.entityAngularName %> = <%- generateTestEntityPrimaryKey(otherEntity.primaryKey) %>;
+                <%_ } _%>
+                        <%= entityInstance %>.<%= reference.name %> = <%= reference.name %>;
+            <%_ } _%>
+                        comp.updateForm(<%= entityInstance %>);
```

Comment:
Can you give input through `ActivatedRoute` and then this functionality is called through `ngOnInit()`.
Method `updateForm` should be protected in component. If you agree to make this change as well in this PR then it would be great. If not then this change can be done in a separate PR.